docs/topics/native/apple-framework.md

```diff
@@ -386,31 +361,32 @@ int main(int argc, const char * argv[]) {
 }
 ```
 
-Here you call Kotlin classes directly from Objective-C code. A Kotlin `object` has the class method 
-function `object`, which allows us to get the only instance of the object and to call 
-`Object` methods on it. 
-The widespread pattern is used to create an instance of the `Clazz` class. You call
-the `[[ DemoClazz alloc] init]` on Objective-C. You may also use `[DemoClazz new]`
-for constructors without parameters.
+Here, you call Kotlin classes directly from Objective-C code. A Kotlin `object` uses the `<object name>.shared` class
+property, which allows you to get the object's only instance and call `Object` methods on it.
+
+The widespread pattern is used to create an instance of the `Clazz` class. You call the `[[ DemoClazz alloc] init]` on
```

Comment:
(Suggestion - up to you)
To create an instance of the `Clazz` class, use `[[ DemoClazz alloc] init]` in Objective-C.

Reply:
it's not an instruction here, but a description of the code above